.github/workflows/tests.yml

```diff
@@ -216,13 +210,13 @@ jobs:
           go-version-file: "go.mod"
       - name: Install AvalancheGo Release
         shell: bash
-        run: BASEDIR=/tmp/e2e-test AVALANCHEGO_BUILD_PATH=/tmp/e2e-test/avalanchego ./scripts/install_avalanchego_release.sh
+        run: ./scripts/run_task.sh install-avalanchego-release
```

Comment:
should we also move this to task?

Reply:
I created a new task to install the avalanchego release to prevent duplication